week04/code.js

```diff
@@ -0,0 +1,83 @@
+const btns = document.getElementById("btns");
+const start = document.getElementById("start");
+const reset = document.getElementById("reset");
+
+const human = document.getElementById("Hchoice");
+const com = document.getElementById("Cchoice");
+
+const score = document.getElementById("score");
+const result = document.getElementById("result");
+
+const elements = ["✌", "✊", "✋"];
+
+let myChoice = null;
+let Cchoice = null;
+
+let myScore = 0;
+let comScore = 0;
+
+const clickStart = () => {
+  document.querySelectorAll("#btns button").forEach((btn) => {
+    btn.removeAttribute("disabled");
+  });
+  myChoice = null;
+  Cchoice = Math.floor(Math.random() * 3);
+
+  let std = 0;
+  const timer = setInterval(() => {
```

Comment:
let으로 timer를 바깥에서 선언했으므로 선언 키워드를 삭제하고 let 변수에 setInterval 함수의 반환 값인 interval ID를 재할당합니다.

```suggestion
  timer = setInterval(() => {
```